Reducing length the doc while keeping the message … (#94)
SpencerDawkins/net-collab-rqmts: draft-kwbdgrr-tsvwg-net-collab-rqmts.md

~~~~~~~~
{: #Figure-netshaper align="center" title="Metadata and Network Shaping"}

{{Figure-netshaper}} shows a bottleneck (access) router on the Server to Client path.
A network shaper in the router manages QoS of multiple users’ flows and can buffer,
discard, or apply other flow control rules. Application layer signaling and feedback
between Client and Server (A - in the figure) adjust transmission rate over several RTTs
using feedback and congestion control algorithms (CCAs). CCAs are generally conservative
and settle to a steady rate to avoid excessive packet loss. In networks where link
conditions (between Client and Router) vary significantly at sub-RTT timescales,
this results in unused bandwidth at short timescales.

> Research (e.g., {{5G-Octopus}}) indicates that media applications can achieve better QoE

Reviewer: ```suggestion
Research (e.g., {{5G-Octopus}}) indicates that media applications can achieve better Quality of Experience (QoE)
```

IETF likes acronym expanded on first use. We don't use QoE later, but it's useful to have **QoE** in the document if someone searches for it.

when sending at a higher rate (less conservative than current CCA) and tolerating some
packet loss or delay of low priority packets. Packet priority and tolerance to delay in
such cases would be provided on-path in a side channel associated with the downstream packet (B).
The requirements for this server-to-network (S2N) metadata are described in {{server-network}}.

The client may provide information to an (access) router to drop lower priority marked packets
of a flow (UDP 4-tuple) temporarily which can in turn allocate available bandwidth
to other flows of that network attachment, especially during a reactive management event.

Network shapers observe flows and apply policies to maximize performancebut are unaware underlying

> flows belinging to the same user and network attachment (e.g., a subscriber connection,a 3GPP PDU

Reviewer: ```suggestion
flows belonging to the same user and network attachment (e.g., a subscriber connection, a 3GPP PDU
```

Session {{net-attach}}). Clients may provide information to an (access) router to drop

> ‘lower priority’-marked packets of a flow (UDP 4-tuple) temporarily during congestion,

Reviewer: ```suggestion
‘lower priority’-marked packets of a flow (UDP 4-tuple) during Reactive Management,
```

> allowing bandwidth allocation to other flows of the same network attachment.

Reviewer: ```suggestion
allowing bandwidth allocation to other packets of the same UDP 4-tuple.
```

## Server-Network Metadata {#server-network}

Application flows (UDP 4-tuple) for live media, eXtended Reality (XR),
and gaming require high bandwidth and low latency. In wireless networks,
some bandwidth cannot be scheduled using feedback-based rate control due
to significant link variations at sub-RTT timescales. Congestion control
algorithms settle to a steady rate to avoid excessive packet loss.
Feedback via ECN/L4S {{?RFC9331}} provides an accurate signal but lacks finer
resolution information of instantaneous bandwidth available.

If application packets can tolerate delay or some loss of lower priority
packets, the network traffic shaper and scheduler can use this information
to provide higher application quality of service. Research in {{5G-Octopus}} indicates
that media applications can achieve better quality when sending at a higher rate
and allowing the network to delay or drop low priority packets.

The metadata in {{relative-priority}} should satisfy constraints identified

> in {{sys-considerations}}. Privacy {{privacy}} requires that metadata should not provide

Reviewer: ```suggestion
in {{sys-considerations}}. Privacy ({{privacy}}) requires that metadata should not provide
```
